nano/node/websocket.hpp

```diff
@@ -136,6 +142,39 @@ namespace websocket
 		friend class session;
 	};
 
+	/**
+	 * Options for payment tracking
+	 */
+	class payment_tracking_options final : public options
+	{
+	public:
+		payment_tracking_options (boost::property_tree::ptree const & options_a);
+
+		/** Validate options */
+		nano::error validate () const;
+
+		/** We keep the subscribe message id so it can be included in notifications. This can be used by clients for message correlation. */
+		std::string id;
+
+		/** The tracked desination account */
+		std::string tracked_account;
+
+		/** Set only if tracking a block */
+		std::shared_ptr<nano::state_block> tracked_block;
```

Comment:
Is it necessary to have this be `shared_ptr`?

Reply:
It was shared with payment_tracking_info, but moving to tracking only hash, so it can become a unique_ptr here and be moved into   👍